src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/config/RedisConfiguration.kt

```diff
@@ -68,7 +68,7 @@ class RedisConfiguration {
     @Value("\${spring.redis.ssl}") ssl: Boolean,
   ): RedLock {
     val scheme = if (ssl) "rediss" else "redis"
-    val passwordString = if (password.isNotEmpty()) "$password@" else ""
+    val passwordString = if (password.isNotEmpty()) ":$password@" else ""
```

Comment:
This confused me at first - I'm assuming there's no user, but it needs to be in the format `user:password` right?

Reply:
that's right yeah